src/Game/Main.java

```diff
@@ -0,0 +1,270 @@
+package Game;
+
+/**
+ * Main class file
+ * @author Angel Lin
+ * 
+ */
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Main {
+    public static void start(String[] args) throws InterruptedException {
+        
+        addLine();
+        //Initialize
+        boolean blattacks = false;
+
+        //Create scanner
+        Scanner scan = new Scanner(System.in);
+
+        //Asks for user input for player name
+        System.out.print("Enter player name: ");
+        String playerName = scan.nextLine();
+        Player mainPlayer = new Player(playerName);
+
+        addLine();
+
+        //Asks for user input for number of entities and items
+        System.out.print("Enter number of entities: ");
+        int intEntityNum = scan.nextInt();
+        System.out.print("Enter number of items: ");
+        int intItemNum = scan.nextInt();
+
+        World start = new World(intEntityNum, intItemNum);
+
+        //Prings out number of entities and items
+        addLine();
+        System.out.println(start);
+        addLine();
+
+        //Create arrays for entities
+        ArrayList<String> entityInfoName = new ArrayList<String>();
+        ArrayList<String> entityInfoNoise = new ArrayList<String>();
```

Comment:
You have an Entity class with info for name and noise, the collection of entities should be maintained in a list of Entity objects, not two separate list for names and noise.  With two lists like you have it here, you defeat the purpose of having a class.